### PR TITLE
remove Logrus with meshkit logger w/signoff

### LINKED ISSUE #11024: Replace usages of logrus with meshkit logger.
### Current Behavior

Currently meshery uses logrus instead of meshkit logger in some places .
### Expected Behavior

Remove all the instances of logrus with meshkit looger

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Jougan-0**
**Notes for Reviewers**

This PR fixes #11024 

Still some instance left I am working on them.
Here is an update pr on the issue.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Deepak0320Singhal** [comment]
@Jougan-0 Thank you for your contribution.
Let's discuss this on Websites's call on Monday. Adding this as an agenda item into the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit).

**Comment by Jougan-0** [comment]
> @Jougan-0 Thank you for your contribution. Let's discuss this on Websites's call on Monday. Adding this as an agenda item into the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit).

This is not a website topic so I don't think we should cover this in website's meeting.
@Deepak0320Singhal

File: server/cmd/error.go [hunk 1]
```diff
@@ -19,6 +19,11 @@ const (
 	ErrClosingDatabaseInstanceCode                = "meshery-server-1011"
 	ErrInitializingRegistryManagerCode            = "meshery-server-1012"
 	ErrInitializingKeysRegistrationCode           = "meshery-server-1013"
+	ErrCreateRegoInstanceCode                     = "replace_me"
+)
+
+var (
+	ErrCreatingRegoInstance = errors.New(ErrCreateRegoInstanceCode, errors.Alert, []string{"Error creating Rego Instance."}, []string{"Unable to create Rego instance, policies will not be evaluated."}, []string{}, []string{})
```
Comment (leecalcote) [inline server/cmd/error.go:26]: I wonder if this shouldn't be "OPA Instance"

File: server/handlers/error.go [hunk 2]
```diff
@@ -136,32 +136,59 @@ const (
...
-	ErrInvalidGenValue     = errors.New(ErrInvalidGenValueCode, errors.Alert, []string{"Invalid value for gen"}, []string{}, []string{}, []string{"please provide a valid value for gen (load generator)"})
-	ErrParseDuration       = errors.New(ErrParseDurationCode, errors.Alert, []string{"error parsing test duration"}, []string{}, []string{"The format of the duration passed could be incorrect"}, []string{"please refer to:  https://docs.meshery.io/guides/mesheryctl#performance-management"})
-	ErrPerformanceTest     = errors.New(ErrPerformanceTestCode, errors.Alert, []string{"Load test error"}, []string{}, []string{"Load test endpoint could be not reachable"}, []string{"Make sure load test endpoint is reachable"})
+	ErrInvalidK8SConfigNil        = errors.New(ErrInvalidK8SConfigNilCode, errors.Alert, []string{"No valid kubernetes config found. Make sure to pass contextIDs in query parameters."}, []string{"Kubernetes config is not initialized with Meshery"}, []string{"Kubernetes config is not accessible to meshery or not valid"}, []string{"Upload your kubernetes config via the settings dashboard. If uploaded, wait for a minute for it to get initialized"})
```
Comment (leecalcote) [inline server/handlers/error.go:150]: Always capitalize Meshery.

Comment (leecalcote) [reply server/handlers/error.go:150]: Always capitalize Kubernetes.

Comment (leecalcote) [reply server/handlers/error.go:150]: Please update this for all errors.

Comment (Jougan-0) [reply server/handlers/error.go]: Done @leecalcote

**Pull request merged**